modules/contacts/hm-contacts.php

```diff
@@ -25,11 +25,24 @@ public function add_contact($data) {
         return true;
     }
 
-    public function get($id, $default=false) {
-        if (!array_key_exists($id, $this->contacts)) {
-            return $default;
+    public function get($id, $email_address="", $default=false) {
```

Comment:
It is not advisable to change signature of a function/method (reorder parameters) unless you made sure you updated all places that call this method.